src/Infrastructure/Persistence/Doctrine/Mapping/User.Organization.orm.xml

```diff
@@ -9,9 +9,6 @@
         <unique-constraint columns="name" name="organization_name" />
     </unique-constraints>
     <many-to-many field="users" inversed-by="organizations" target-entity="App\Domain\User\User">
-        <cascade>
```

Comment:
Ici, je supprime la cascade qui n'apporte rien (aucune diff dans les Doctrine migrations). Cf https://www.doctrine-project.org/projects/doctrine-orm/en/2.14/reference/association-mapping.html#many-to-many-bidirectional